Memory/Hooks.cpp

```diff
@@ -250,10 +250,10 @@ void Hooks::Init() {
 		// Mob::_jumpFromGround
 		void* jump = reinterpret_cast<void*>(FindSignature("48 89 5C 24 10 57 48 83 EC ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 44 24 ?? 48 8B 19 48 8D"));
 		g_Hooks.JumpPowerHook = std::make_unique<FuncHook>(jump, Hooks::JumpPower);
-
+		//ServerInstance::onAppSuspended (maybe)
```

Comment:
below it says MinecraftGame and not serverinstance

Reply:
thanks, the devs can change it if they need toa